contracts/carbon/Strategies.sol

```diff
@@ -333,7 +334,7 @@ abstract contract Strategies is Initializable {
             } else if (newOrders[i].y > orders[i].y) {
                 // liquidity increased - deposit the difference
                 uint128 delta = newOrders[i].y - orders[i].y;
-                _validateDepositAndRefundExcessNativeToken(token, owner, delta, value);
+                _validateDepositAndRefundExcessNativeToken(token, owner, delta, value, true);
```

Comment:
Technically speaking, we don't need it at this point, because the strategy creation would have failed.
It's not a 100% thing, since the token contract may have been upgraded to include transfer-fee (or perhaps even that fee mechanism "decides to not take a fee" on strategy creation).
But I'd say it's a 99.9999% thing, so your call...

Reply:
It's definitely possible to have fee-on-transfer activated later. Additionally, it's possible to have strategies with fee-on-transfer tokens created before the upgrade. imo it's worth it